coremltools/models/utils.py

```diff
@@ -1675,3 +1681,394 @@ def randomize_weights(mlmodel: "_ct.models.MLModel"):
     )
 
     return randomized_mlmodel
+
+
+def bisect_model(
+    model_path: str,
+    output_dir: str,
+    remove_original: _Optional[bool] = False,
+    merge_chunks_to_pipeline: _Optional[bool] = False,
+    check_output_correctness: _Optional[bool] = False,
+):
+    """
+    Utility function to split a mlpackage model into two mlpackages of approximately same file size.
+
+    The implementation of this util is mainly migrated from the below source:
```

Comment:
I think we can remove this line, since we would be eventually deleting that file, or making it a very thin wrapper on top of this

Reply:
I agree. Let's remove this.

Reply:
Done